test/system/120-load.bats

```diff
@@ -89,11 +89,6 @@ verify_iid_and_name() {
         skip "impossible due to pitfalls in our SSH implementation"
     fi
 
-    # See https://github.com/containers/podman/pull/21431
-    if [[ -n "$PODMAN_IGNORE_CGROUPSV1_WARNING" ]]; then
-        skip "impossible to test due to pitfalls in our SSH implementation"
-    fi
-
```

Comment:
@edsantiago PTAL [this `sys podman debian-13 root host sqlite` failure](https://api.cirrus-ci.com/v1/artifact/task/4596769458225152/html/sys-podman-debian-13-root-host-sqlite.log.html) is curious.  ISTM like `PODMAN_IGNORE_CGROUPSV1_WARNING` being forced 'true' was hiding this failure somehow.

The test error is coming from the end of `run_podman()`, but the condition catching it is really complex.  Since the rootless version passed, should I add a special-case in `setup_environment()` on rootful Debian, where `setup_rootless()` enables lingering for `$ROOTLESS_USER` (like the warning message suggests)?